### ISSUE TITLE
C# (dotnet core) version

### ISSUE THREAD

**Issue by genez** [spion/hashtable-latencies#14]
Hello,
if you are interested I can submit a C# version that can run on dotnet core

I am prototyping something similar (but with TCP and UDP connections) and I have the same issues (high latency with large maps, in special way in Go)

I should have something in Java, too

**Comment by spion** [comment]
Yes, both the Java and the C# versions would be a welcome addition
